TestPlaybooks/playbook-TestIsMaliciousIndicatorFound.yml

```diff
@@ -0,0 +1,413 @@
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 480,
+          "y": 545
+        }
+      }
+  "8":
+    id: "8"
+    taskid: c6b5e42f-3366-46ec-8df3-8146ef0c33a9
+    type: condition
+    task:
+      id: c6b5e42f-3366-46ec-8df3-8146ef0c33a9
+      version: -1
+      name: IsMaliciuosFound-no-suspicious
+      scriptName: IsMaliciousIndicatorFound
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      "no":
+      - "11"
+      "yes":
+      - "10"
+    scriptarguments:
+      includeSuspicious: {}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 480,
+          "y": 720
+        }
+      }
+  "10":
+    id: "10"
+    taskid: 6f22921c-07e1-4315-83ab-5cbb9013a6c2
+    type: regular
+    task:
+      id: 6f22921c-07e1-4315-83ab-5cbb9013a6c2
+      version: -1
+      name: Fail
+      description: |-
+        Verifies path in context:
+        - Verifies path existence
+        - If matching object is an array: verify fields exists in each of the objects in the array
+        - If matching object is not an array: verify fields exists in matching object
+        - if 'expectedValue' is given: ensure that the given value is equal to the context path
+      scriptName: VerifyContext
+      type: regular
+      iscommand: false
+      brand: ""
+    scriptarguments:
+      expectedValue: {}
+      fields: {}
+      path:
+        simple: xxx
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 265,
+          "y": 895
+        }
+      }
+  "11":
+    id: "11"
+    taskid: bb8f1bb0-23c3-490c-8b49-6a32ceae65af
+    type: condition
+    task:
+      id: bb8f1bb0-23c3-490c-8b49-6a32ceae65af
+      version: -1
+      name: IsMaliciuosFound-with-suspicious
+      scriptName: IsMaliciousIndicatorFound
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      "no":
+      - "12"
+      "yes":
+      - "4"
+    scriptarguments:
+      includeSuspicious:
+        simple: "yes"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 695,
+          "y": 895
+        }
+      }
+  "12":
+    id: "12"
+    taskid: 8877482e-d76f-48e9-8d7e-4943245a3338
+    type: regular
+    task:
+      id: 8877482e-d76f-48e9-8d7e-4943245a3338
+      version: -1
+      name: Fail
+      description: |-
+        Verifies path in context:
+        - Verifies path existence
+        - If matching object is an array: verify fields exists in each of the objects in the array
+        - If matching object is not an array: verify fields exists in matching object
+        - if 'expectedValue' is given: ensure that the given value is equal to the context path
+      scriptName: VerifyContext
+      type: regular
+      iscommand: false
+      brand: ""
+    scriptarguments:
+      expectedValue: {}
+      fields: {}
+      path:
+        simple: xxx
```

Comment:
I would change text here to imply which task failed (something like "Did not found malicious indicator but it exists..")